engine/chunk/build.gradle

```diff
@@ -7,13 +7,13 @@ description 'Engine Chunks: Array-like data structures for dense, efficient data
 
 dependencies {
     api project(':Util')
-    implementation project(':Base'),
-            project(':codec-base:')
+    implementation project(':Base')
+    implementation project(':codec-api:')
```

Comment:
Looks like this should be api, given it's publicly exposed on `ByteChunk`. That said, it doesn't look like anything calls `io.deephaven.chunk.ByteChunk#applyDecoder(io.deephaven.util.codec.ObjectDecoder<T>)`... should we prefer to get rid of this unused method and remove dependency @rcaudy ? (Would be happy to leave as issue for follow-up as well.)

Reply:
it's from some legacy thing.  I have no use for it